### ISSUE TITLE
Berlin HF status?

### ISSUE THREAD

**Issue by artob** [rust-blockchain/evm#12]
Grepping through the code base, I found no mention of the [Berlin hardfork](https://eips.ethereum.org/EIPS/eip-2070), nor the EIPs included in that hardfork.

What are your plans in that regard? Has anyone as yet began work on Berlin HF support, is that perhaps in a personal fork somewhere? Would contributions in that regard be welcome?

**Comment by artob** [comment]
Related pull requests and issues:
- https://github.com/rust-blockchain/evm/pull/14
- https://github.com/paritytech/frontier/issues/292

**Comment by trmaphi** [comment]
@artob, not sure this what you're looking for, however, I found a source code that relates to all fork spec https://github.com/rust-blockchain/evm-tests/blob/360d1cbeb8716fe6fc77a713adf096353fd0df1e/ethjson/src/spec/spec.rs#L44

**Comment by artob** [comment]
@trmaphi Yes, the Berlin HF work is underway now, though not yet merged: #14

**Comment by eugene-babichenko** [comment]
Now that it is merged, can we see a release anytime soon?
